File: Books.java
public class Books {
    private int ID;
    private String title;
    private String author;
    private int year;
    private boolean rented = false;

    public Books(int ID, String title, String author, int year) {
        this.ID = ID;
        this.title = title;
        this.author = author;
        this.year = year;
        this.rented = rented;
    }
    //Getters y Setters de la clase
    public int getID() {
        return ID;
    }

    public String getTitle() {
        return title;
    }
    public void setTitle(String title) {
        this.title = title;
    }
    public String getAuthor() {
        return author;
    }
    public void setAuthor(String author) {
        this.author = author;
    }
    public int getYear() {
        return year;
    }
    public void setYear(int year) {
        this.year = year;
    }
    public boolean isRented() {
        return rented;
    }
    public void setRented(boolean rented) {
        this.rented = rented;
    }
}

File: Books.py
class Books:
    def __init__(self, ID, title, author, year):
        self.ID = ID
        self.title = title
        self.author = author
        self.year = year
        self.rented = False

    def get_ID(self):
        return self.ID

    def get_title(self):
        return self.title

    def set_title(self, title):
        self.title = title

    def get_author(self):
        return self.author

    def set_author(self, author):
        self.author = author

    def get_year(self):
        return self.year

    def set_year(self, year):
        self.year = year

    def is_rented(self):
        return self.rented

    def set_rented(self, rented):
        self.rented = rented